packages/gatsby-theme/src/components/Grid/Grid.stories.mdx

```diff
@@ -0,0 +1,139 @@
+import { Meta, Story, Props, Preview } from "@storybook/addon-docs/blocks"
+import { withA11y } from "@storybook/addon-a11y"
```

Comment:
It looks good but I don't see any decorator invoking `a11y`, this is expected?
if not we should probably remove it, and adding in a future change :)

Reply:
@omero This is expected, `withA11y` is a Storybook addon and this definition will be reflected in the Storybook interface.